packages/epk-cypher/test/ethereum-private-key-cypher-provider.test.ts

```
@@ -0,0 +1,242 @@
+  encryptionParams: {
+    key: '299708c07399c9b28e9870c4e643742f65c94683f35d1b3fc05d0478344ee0cc5a6a5e23f78b5ff8c93a04254232b32350c8672d2873677060d5095184dad422',
+    method: EncryptionTypes.METHOD.ECIES,
+  },
+  identity: {
+    type: IdentityTypes.TYPE.ETHEREUM_ADDRESS,
+    value: '0xaf083f77f1ffd54218d91491afd06c9296eac3ce',
+  },
+  privateKey: '0x04674d2e53e0e14653487d7323cc5f0a7959c83067f5654cafe4094bde90fa8a',
+  publicKey:
+    '299708c07399c9b28e9870c4e643742f65c94683f35d1b3fc05d0478344ee0cc5a6a5e23f78b5ff8c93a04254232b32350c8672d2873677060d5095184dad422',
+};
```

Biome

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)